.github/workflows/sangertest.yml

```diff
@@ -1,18 +1,12 @@
 name: nf-core Sanger LSF tests
 
 on:
-  pull_request:
```

Comment:
I think it will be enough just comment out this line, not the other parts. and later we can put them back very quickly.

Reply:
if not, just remember to put everything back next time.